Team/sultans of swing/fix/door interaction

smiljanaspasic opened the pull request.

smiljanaspasic committed:
small fixes to character controller

smiljanaspasic committed:
improved door interaction

DPrivalov commented on source/dagger/gameplay/team_game/door_interaction.cpp:
```diff
@@ -15,52 +15,58 @@ using namespace team_game;
+			pickedup = true;
 			Engine::Registry().destroy(keyentity);
 		}
+		else { pickedup = false; }
```
try to follow some codestyle:
put "{" after new line or not, decide this and then do it in one way. 

I see different ways inside one file

DPrivalov commented on source/dagger/gameplay/team_game/door_interaction.cpp:
```diff
@@ -15,52 +15,58 @@ using namespace team_game;
 
-bool keytaken = false;
 
+Bool pickedup = false;
```
can't u put it inside DoorSystem at least? as a member field

smiljanaspasic replied:
Thank you for all of the nice suggestions 😃

DPrivalov commented on source/dagger/gameplay/team_game/team_game_main.cpp:
```diff
@@ -116,78 +116,83 @@ void SetupWorldJovica(Engine& engine_, Registry& reg_)
 void SetupWorldSmiljana(Engine& engine_, Registry& reg_) {
 
-/*
+    /*
```
too many white spaces/new lines, clean it up a bit before merging.

smiljanaspasic committed:
code cleanup

also added that the player cannot escape the door if he's not in the hero shape
